src/fixed.ts

```diff
@@ -61,6 +63,8 @@ export default class FixedThreadPool<Data = any, Response = any> {
     if (!isMainThread) {
       throw new Error('Cannot start a thread pool from a worker thread !!!')
     }
+    // TODO christopher 2021-02-07: Improve this check e.g. with a pattern or blank check
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
     if (!this.filePath) {
```

Comment:
If you use `typeof this.filePath === undefined`, that would fix the linter.

Reply:
And fail the test... yeah...
It's not the same

Please try 
```js
!'' // true
typeof '' === undefined // false
```